sdks/python/src/honcho/async_client/client.py

```
@@ -0,0 +1,317 @@
+            An AsyncPeer object that can be used to send messages, join sessions, and
+            query the peer's knowledge representations
+
+        Raises:
+            ValidationError: If the peer ID is empty or invalid
+        """
+        if config:
+            return await AsyncPeer.create(
+                id, self.workspace_id, self._client, config=config
+            )
+        else:
+            return AsyncPeer(id, self.workspace_id, self._client)
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)